lib/dialects/mssql/index.js

```diff
@@ -350,25 +360,25 @@ class Client_MSSQL extends Client {
             );
           }
 
-          return Driver.TYPES.BigInt;
+          return [binding, Driver.TYPES.BigInt];
         }
 
-        return Driver.TYPES.Int;
+        return [binding, Driver.TYPES.Int];
       }
       default: {
         // if (binding === null || typeof binding === 'undefined') {
         //   return tedious.TYPES.Null;
         // }
```

Comment:
can we remove this comment (not in your PR, but can be good to clean up)

Reply:
> Why only MsSQL dialect ?

Each dialect within knex has it's own sauce for how to go from nodejs types to database specific types. Some even defer to the external library for handling the mapping itself (AFAIK). Executing this PR for all dialects would require each knex dialect/client to own this mapping itself. Would need someone that knows how to do that mapping for the other dialects to help take that on. I'm not sure this is as big of an issue with other dialects either.

> can we remove this comment (not in your PR, but can be good to clean up)
Will do.